### PR TITLE
Fix typo in StringSet

### PR DESCRIPTION
**Pull request by zetashift**

### CODE REVIEW

File: modules/docs/src/02-the-smithy-idl/01-smithy-idl.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ list IntList {
 }
 
 set StringSet {
-  member: Set
```
Comment (kubukoz) [inline modules/docs/src/02-the-smithy-idl/01-smithy-idl.md:63]: **thought:** lol

**Review by kubukoz — APPROVED** [review]

**Comment by Baccata** [comment]
how are there conflicts in this PR ? :/

**Comment by daddykotex** [comment]
> how are there conflicts in this PR ? :/

My take is that the PR was created off `main` in the zetashift fork, but targets series/0.17 here.

**Comment by zetashift** [comment]
> > how are there conflicts in this PR ? :/
> 
> My take is that the PR was created off `main` in the zetashift fork, but targets series/0.17 here.

Ah yup I think this is what happened. i was just mindlessly clicking through githubs online editor ><.
I'll just make the right PR locally!

**Pull request closed without merging**